report-ng/app/src/app.ts

```diff
@@ -80,12 +80,6 @@ export class App {
                 name: "tests",
                 title: 'Tests'
             },
-            // {
-            //     route: 'threads',
```

Comment:
How do you reach the Thread view now?

Reply:
The Thread view is reached in the exact same way as before (line 118). That was just a duplicated code fragment I removed.